features/import.feature

```diff
@@ -342,3 +342,50 @@ Feature: cli-kintone import command
     When I run the command with args "record import --base-url $$TEST_KINTONE_BASE_URL --app $APP_ID --api-token $API_TOKEN --attachments-dir ./attachments --file-path 9f56a2be-c8f0-4ecc-8f62-b5b430e34e25.csv"
     Then I should get the exit code is non-zero
     And The output error message should match with the pattern: "Error: ENOENT: no such file or directory"
+
+  Scenario: CliKintoneTest-51 Should import the records successfully with --fields specified.
+    Given The app "app_for_import" has no records
+    And The csv file "CliKintoneTest-51.csv" with content as below:
```

Comment:
@tuanphamcybozu 

In the CSV file, we should have one more column, such as Number. So that, the test case can verify that even if the CSV has more fields, It should only import the specified fields.

Reply:
@hung-cybo 
I added one more column. Please check it.